contracts/SnappAuction.sol

```diff
@@ -149,34 +151,44 @@ contract SnappAuction is SnappBase {
         emit StandingSellOrderBatch(currentBatchIndex, accountId, buyTokens, sellTokens, buyAmounts, sellAmounts);
     }
 
-    function placeSellOrder(
-        uint8 buyToken,
-        uint8 sellToken,
-        uint128 buyAmount,
-        uint128 sellAmount
-    ) public onlyRegistered() {
-
-        if (
-            auctionIndex == MAX_UINT ||
-            auctions[auctionIndex].size == maxUnreservedOrderCount() ||
-            block.timestamp > (auctions[auctionIndex].creationTimestamp + 3 minutes)
-        ) {
-            createNewPendingBatch();
-        }
-
-        // Update Auction Hash based on request
+    function placeSellOrders(bytes memory packedOrders) public onlyRegistered() {
```

Comment:
Unfortunately not. That was the reason for the large contract size that wouldn't migrate. Do you still think is should be left here?

Reply:
That is to say, if we leave the old interface, then the contract won't migrate without optimization enabled. Is this what we really want?

Reply:
I see. I believe we should mitigate the large contract size for now by enabling optimisation (or even increasing the block gas limit on ganache), since otherwise we will reach the limit in the next PR again.

I will make it my priority to refactor the SnappBase into a library to give us a bit more breathing space long term for deploying within the gas limit.